backend/handlers/createSession.js

```diff
@@ -0,0 +1,44 @@
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+const createSession = ( async (request, response) => {
```

Comment:
are we not using express-session? We shouldn't have to manually implement session creation logic if we're using that package.

Reply:
ah this is not creating a session -- it's loggin in right? we can keep we should just name it more clearly. I thought you were trying to create a session.

Reply:
my bad for the bad naming. i renamed it to login handler. whoops